src/components/OSCALControlModification.js

```diff
@@ -42,13 +42,30 @@ function getAlterAddsOrRemovesDisplay(addsElements, addsLabel, controlPartId) {
         Name: {item.name}, Value: {item.value}
       </Typography>
     ));
+  let removedTypographies = null;
+  if (addsLabel === "Removes ") {
+    removedTypographies = addsElements
+      .flatMap((elt) => elt ?? [])
+      .map((y) => (
```

Comment:
I dont think we need to shorten `element` here. Can we also change `y` to something more descriptive like `item`?

Reply:
Thanks for your suggestions, I will adjust those labels accordingly.